Mu2eG4/geom/STM_v08.txt

```diff
@@ -22,6 +22,8 @@ bool   stm.SScollimator.liner.build   = false;
 bool   stm.detector.stand.build       = false;
 bool   stm.shield.build               = true;
 
+//STM reference point that was previously calculated automatically based on the CRV-D location
+double stm.referenceZ = 18766.81;
```

Comment:
Just to ask how you got the number: did you just print out the value of ```z_crv_max``` in ```STMMaker```, or something else?

Reply:
Yes, that's exactly what I did. I printed out the value and used it in the txt file.